rules/tests/BUILD

```diff
@@ -0,0 +1,7 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+load("//rules/tests:const_unittest.bzl", "const_test_suite")
+
+const_test_suite()
```

Comment:
Do we have to do anything to make this run in CI?

Reply:
Nope - they are included by the [Earlgrey SW Build and Test job](https://dev.azure.com/lowrisc/opentitan/_build/results?buildId=116075&view=logs&j=5df5adf8-3577-5927-f0b2-31afed2697ee&t=1253ba27-4e8c-57af-d0ba-888237644875&l=361).

Reply:
Great! Still LGTM.